optimum/intel/openvino/modeling_visual_language.py

```diff
@@ -287,6 +299,18 @@ def __init__(
         except AttributeError:
             pass
 
+    def clear_requests(self):
```

Comment:
I suggest to move compilation, nameing related features that are not directly relate to quantization into this pr https://github.com/huggingface/optimum-intel/pull/967
part of them already fixed, so I can join additional changes provide from your side too and provide tests for basic api

Reply:
@eaidova This is fine with me. But we'll have to merge #967 before this PR.

Reply:
just to not bring complexity, Let's merge your changes fist, then I'll update remained part in my PR